packages/e2e-test-utils/src/index.js

```diff
@@ -35,6 +35,10 @@ export { installPlugin } from './install-plugin';
 export { isCurrentURL } from './is-current-url';
 export { isInDefaultBlock } from './is-in-default-block';
 export { loginUser } from './login-user';
+export {
+	multiSelectBlocksByIds,
```

Comment:
Could we consolidate those two methods into one API? Those methods are exposed as public API and added to the documentation. We will have to maintain them for long because the 3rd party project will use it.

Reply:
I was talking with @mcsf about making the e2e tests more `user` interactive and not multi select `programmatically`, so I'll remove them as they will not be used. Do you think it has any value to keep them for now, if not used?

Reply:
Not at all, we shouldn't keep methods that aren't used (unless they were part of public API previously).